packages/bitcore-node/src/modules/ripple/api/csp.ts

```diff
@@ -262,12 +279,17 @@ export class RippleStateProvider extends InternalStateProvider implements CSP.IC
         network,
         chain: this.chain,
         txid: tx.transaction.hash,
+        from: tx.transaction.Account,
         blockHash: (tx as any).ledger_hash || '',
         blockHeight: tx.ledger_index || -1,
         blockTime: new Date(),
         blockTimeNormalized: new Date(),
-        value: Number(tx.transaction.Amount),
-        fee: Number(tx.transaction.Fee),
+        value: Number(tx.transaction.Amount) * 1e6,
```

Comment:
how will the `* 1e6` impact things already consuming this

Reply:
The only thing would be the events. I'm checking this